test/Test/WebDriver/ChromeDriverNonW3CTest.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/**
+ * Copyright 2021-2022 Anthon Pang. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @package WebDriver
+ *
+ * @author Anthon Pang <apang@softwaredevelopment.ca>
+ */
+
+namespace Test\WebDriver;
+
+use Test\WebDriver\WebDriverTestBase;
+use WebDriver\Browser;
+use WebDriver\Session;
+
+/**
+ * ChromeDriver
+ *
+ * @package WebDriver
+ *
+ * @group Functional
+ */
+class ChromeDriverNonW3CTest extends WebDriverTestBase
```

Comment:
Should this extend ChromeDriverTest instead?

Reply:
Fixed.